wordpress.yaml

```diff
@@ -1,7 +1,81 @@
 ---
 - name: Deploy Wordpress
-  hosts: wordpress
+  hosts: all
+  become: yes
   tasks:
+    - name: Update system
+      apt:
+        update_cache: yes
 
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna setup Wordpress application'
+    - name: Upgrade system
+      apt:
+        upgrade: dist  
+
+    - name: Install software 
+      apt:
+        name: 
+          - php-mysql 
+          - mysql-server 
+          - php-curl 
+          - php-gd 
+          - php-intl 
+          - php-mbstring 
+          - php-soap 
+          - php-xml 
+          - php-xmlrpc 
+          - php-zip 
+          - nginx 
+          - php7.4-fpm
+
+    - name: Install wp-cli 
+      get_url:
+        url: https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+        dest: /usr/local/bin/wp
+        mode: '0755'
+    
+    - name: Download WP
+      get_url: 
+        url: https://wordpress.org/latest.zip
+        dest: /var/www
+    - name: Extract wp
+      unarchive: 
```

Comment:
when you have many users, replaceing working files to original ones would be noticed. How can it be fixed?